distributor/cmd/main.go

```diff
@@ -442,39 +442,60 @@ func pollHTTPEventSource(endpoint string, token string, topics []string) {
 	}
 }
 
+// pollEventsForTopic polls .triggered events from the Keptn api, and forwards them to the receiving service
 func pollEventsForTopic(endpoint string, token string, topic string) {
 	logger.Infof("Retrieving events of type %s", topic)
 	events, err := getEventsFromEndpoint(endpoint, token, topic)
 	if err != nil {
 		logger.Errorf("Could not retrieve events of type %s from endpoint %s: %v", topic, endpoint, err)
 	}
 	logger.Infof("Received %d new .triggered events", len(events))
+
+	// ensure CloudEvents Cache exists
+	if ceCache == nil {
```

Comment:
I think you could even move the variable declaration to the top in line 69 and remove this check. It's a global variable.

Reply:
`ceCache` is already initialized in `createHTTPConnection`, so I guess we don't need the check here.
```
func createHTTPConnection() {
	if env.PubSubRecipient == "" {
		logger.Error("No pubsub recipient defined")
		return
	}

	ceCache = lib.NewCloudEventsCache()

	eventEndpoint := getHTTPPollingEndpoint()
	topics := strings.Split(env.PubSubTopic, ",")

	pollingInterval, err := strconv.ParseInt(env.HTTPPollingInterval, 10, 64)
	if err != nil {
		pollingInterval = defaultPollingInterval
	}

	pollingTicker := time.NewTicker(time.Duration(pollingInterval) * time.Second)

	for {
		<-pollingTicker.C
		pollHTTPEventSource(eventEndpoint, env.KeptnAPIToken, topics)
	}
}
```

Reply:
I think we need it for running unit tests on the `pollEventsForTopic` function.

Reply:
I've moved it.